src/test/java/org/isf/admission/rest/AdmissionControllerTest.java

```diff
@@ -173,43 +173,21 @@ public void testGetCurrentAdmission_200() throws Exception {
 
 		Patient patient = PatientHelper.setup();
 		when(patientManagerMock.getPatientById(patientCode))
-				.thenReturn(patient);
+						.thenReturn(patient);
 
 		Admission admission = AdmissionHelper.setup();
 		when(admissionManagerMock.getCurrentAdmission(patient))
-				.thenReturn(admission);
+						.thenReturn(admission);
 
 		MvcResult result = this.mockMvc
-				.perform(get(request)
-						.param("patientCode", patientCode.toString())
-						.contentType(MediaType.APPLICATION_JSON)
-				)
-				.andDo(log())
-				.andExpect(status().is2xxSuccessful())
-				.andExpect(status().isOk())
-				.andExpect(content().string(containsString(AdmissionHelper.asJsonString(admissionMapper.map2DTO(admission)))))
-				.andReturn();
-
-		LOGGER.debug("result: {}", result);
-	}
-
-	@Test

```

Comment:
I don't think this test should be removed. There are 3 forms of `/admissions/admittedPatients`; the first is to get all patients without search terms and dates; the second is to get all patients with search terms; and the third is to get all patients with search terms and dates.   If this test is removed the first form is not tested.  It can be made to meet that test type by the following changes.